crates/runtime/src/dataconnector/graphql.rs

```diff
@@ -576,18 +576,17 @@ impl GraphQL {
                 message: "Invalid URL in dataset `from` definition",
             },
         )?;
-        let json_path = self
+        let json_pointer = self
             .params
-            .get("json_path")
-            .ok_or("`json_path` not found in params".into())
+            .get("json_pointer")
+            .ok_or("`json_pointer` not found in params".into())
```

Comment:
Since this is a breaking change, some tests should've been broken if there is any? If not, it might be good to consider adding some.

Reply:
Yes I fixed the test.  Updated.